apps/system/js/app_usage_metrics.js

```diff
@@ -558,18 +538,18 @@
 
     // Is there data to be sent and is this an okay time to send it?
     if (!this.metrics.isEmpty() && this.idle && navigator.onLine) {
-      var absoluteTime = Date.now();
+      var relativeTime = peformance.now();
       // Have we tried and failed to send it before?
-      if (this.lastFailedTransmission > this.metrics.startTime()) {
+      if (this.lastFailedTransmission > this.metrics.relativeStartTime) {
 
         // If so, then send it if the retry interval has elapsed
-        if (absoluteTime - this.lastFailedTransmission > AUM.RETRY_INTERVAL) {
+        if (relativeTime - this.lastFailedTransmission > AUM.RETRY_INTERVAL) {
           this.transmit();
         }
       }
       // Otherwise, if we have not failed to transmit, then send it if the
       // reporting interval has elapsed.
-      else if (absoluteTime - this.metrics.startTime() > AUM.REPORT_INTERVAL) {
+      else if (totalElapsed > AUM.REPORT_INTERVAL) {
```

Comment:
This will only send the report after two weeks of the user having the phone turned on. If a user turns off their phone every night and only has it on for 12 hours a day, then their reports will only be sent every 4 weeks.

I think you are going to have to use absolute time for at least some of this module.